scss/_patterns_navigation.scss

```diff
@@ -651,6 +651,174 @@ $spv-navigation-logo-bottom-position: 0.125rem; // 2px when 1rem is 16px
     }
   }
 
+  @keyframes fade-in {
```

Comment:
Would be good to namespace it a bit to avoid potential conflicts, something like `vf-nav-fade-it` ?